lua/lvim/config/_deprecated.lua

```diff
@@ -114,6 +114,13 @@ function M.post_load()
       requires = "dependencies",
     }
 
+    alternatives.tag = function()
+      if spec.tag == "*" then
+        spec.version = "*"
+        return [[version = "*"]]
+      end
+    end
```

Comment:
this seems very specific, was it ever used before? or what's the context here? I would've expected `tag = ""` if anything

Reply:
I forgot to remove `tag = "version"` from the table above, done now

from the lazy's migration guide: 
> tag='\*' -> version="\*"

other bullet points don't have an equal sign, so `version` should be used instead of `tag` only in the `tag == "*"` case